libs/sql-parser/src/main/java/io/crate/sql/parser/SqlParser.java

```diff
@@ -78,23 +79,40 @@ public static List<Statement> createStatements(String sql) {
         return ((MultiStatement) INSTANCE.generateStatements(sql)).statements();
     }
 
+    public static List<Statement> createStatementsForSimpleQuery(String sql, Function<String, Object> stringLiteralParser) {
```

Comment:
I think the `createStatements` above could be removed. Invoking multiple statements at once is afaik only possible in simple query mode. Not via extended query mode or HTTP endpoint

Reply:
It's uses in the parser tests, but I adjusted them to use the new function instead.